### DISCUSSION TITLE
Asking for help about how to contribute

### DISCUSSION THREAD

**Discussion by WizardHowlhaha** [rhinstaller/anaconda#5517] [Q&A]
I am using the Centos Stream 9, and the corresponding anaconda version is anaconda-34.
When I try to built my own Linux distribution with lorax and genisoimage (The difference from Centos Stream is that in my iso I added an additional local repo of my own in addition to the AppStream)，and when I entered the installation interface and clicked the `Installation Source` button, an index error was triggered, view the traceback in /tmp/anaconda.log as follows : 
![image](https://github.com/rhinstaller/anaconda/assets/93726790/8235b0a9-4a4d-4ba0-8b52-7cde6cb55b23)
Now I have solved the issue and made a patch,I learned through google that centos stream 9 is from fedora-34, so I submitted this commit to the f34-devel branch, and made this pull request:
https://github.com/rhinstaller/anaconda/pull/5516
But I don't know if I did it right, and also there are still some failed check items, can anybody give some help?

**Comment by jstodola** [comment]
Hello, since the change is needed in CentOS Stream 9, it should go to the `rhel-9` branch, see the official documentation:
https://anaconda-installer.readthedocs.io/en/latest/contributing.html#which-is-my-target-git-branch

Please note that in order to get any change to CentOS Stream/RHEL, you should also report the problem as a bug in https://issues.redhat.com/projects/RHEL against `anaconda`.

**Reply by WizardHowlhaha** [reply]
Got it.Thanks
